store/src/java/com/zimbra/cs/mailbox/MailboxOperation.java

```diff
@@ -111,7 +111,8 @@ public enum MailboxOperation {
     SetDisableActiveSync(91),
     SetWebOfflineSyncDays(92),
     DeleteConfig(93),
-    View(94);
+    View(94),
+    SetPop3Uid(95);
```

Comment:
this is for ZCS-12967

Reply:
we should remove `SetPop3Uid(95)` from this as this will be taken care in ZCS-12967 and also for cloud this ID(95) will be different there.

Reply:
done.